apps/asap-cli/package.json

```diff
@@ -22,6 +22,7 @@
     "csv-parse": "^4.12.0",
     "got": "^11.6.0",
     "jwt-encode": "^1.0.1",
+    "luxon": "^1.26.0",
```

Comment:
TIL the BE already uses luxon in some places. Bit unfortunate that we have an inconsistency here (FE uses date-fns) but no biggie

Reply:
Yeah we need to mock the time in some place. This makes it easier